cmd/krew/cmd/search.go

```diff
@@ -118,6 +121,53 @@ Examples:
 	PreRunE: checkIndex,
 }
 
+func searchByNameAndDesc(keyword string, target searchTarget) []string {
+	if keyword == "" {
+		return target.names
+	}
+
+	var searchResults = make([]string, 0, len(target.names))
+
+	// find by names
+	matches := fuzzy.Find(keyword, target.names)
+	searchResultsMap := make(map[int]bool)
+	for _, m := range matches {
+		searchResults = append(searchResults, m.Str)
+		searchResultsMap[m.Index] = true
+	}
+
+	// find by short descriptions
+	fuzzyMatches := fuzzy.Find(keyword, target.descriptions)
+	for _, m := range fuzzyMatches {
+		// use map to deduplicate and score > 0 to match more accurately
+		if _, exist := searchResultsMap[m.Index]; !exist && isMatchContinuous(m.MatchedIndexes) {
+			searchResults = append(searchResults, target.names[m.Index])
+		}
+	}
+
+	return searchResults
+}
+
+func isMatchContinuous(matchIndexes []int) bool {
```

Comment:
can you explain what this does and what is the "continuous" check is for? 
ideally methods like these warrant tests. :)

Reply:
I now see you're looking at whether the fuzzy match occurs continuously.
May I ask why did you do that? If you do that, that is no longer becomes "fuzzy" search? 
The intent is also not clear from the code (and one has to really pay attention to see that) --or the tests?

Reply:
Deleted.
The reason is after we use fuzzy search, the matched result may be not continuous words.
So use `m.Score > 0` to make it more accurately.